main.html

```diff
@@ -0,0 +1,151 @@
+<!DOCTYPE html>
+
+<html>
+	<head>
+		<meta charset="UTF-8">
+		<title>pokedex</title>
+		<meta name="viewport" content="width=device-width, initial-scale=1">
+		<link rel="stylesheet" type="text/css" href="main.css">
+		<link rel="stylesheet" media="(max-width: 800px) and (min-width: 401px)" href="ipad.css">	
+		<link rel="stylesheet" media="(max-width: 400px) and (min-width: 1px)" href="iphone.css">
```

Comment:
the min-width: 1px statement for iphone css is unnecessary. the stylesheet will take over once the screen-width is smaller than 400px